server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/sql/MessageServiceParser.java

```diff
@@ -221,4 +262,14 @@ private String getMessageForParser(String messageKey, String... parameters) {
         .getMessageService()
         .getMessage(messageKey, (Object[]) parameters);
   }
+
+  private Integer tryParseInt(String input) {
+    Integer parsedValue;
+    try {
+      parsedValue = Integer.parseInt(input);
+    } catch (Exception ex) {
+      parsedValue = null;
+    }
+    return parsedValue;
+  }
```

Comment:
```suggestion
 private Integer tryParseInt(String input) {
    try {
        return Integer.parseInt(input);
    } catch (NumberFormatException ex) {
        return null;
    }
}
```

Reply:
This was copied over from another implicit dialect's MessageServiceParser.